src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -77,6 +92,14 @@ private ExtensionSettings readExtensionSettings() throws IOException {
         return extensionSettings;
     }
 
+    // current placeholder for extension point override getNamedWriteables()
+    private List<NamedWriteableRegistry.Entry> getNamedWriteables() {
+
+        List<NamedWriteableRegistry.Entry> namedWriteables = new ArrayList<>();
+        // extensions will add named writeable registry entries here
```

Comment:
Let's follow the javadoc style here for the comments
```suggestion
        // Extensions will add named writeable registry entries here
```

Reply:
Also, is this part incomplete as we are creating an empty ArrayList and returning? Am I missing something here?

Reply:
I'll convert this to an api that the Extension can use to pass in their named writeable entries, as per Sarat's comment